foreman_discovery.gemspec

```diff
@@ -3,6 +3,12 @@ $:.push File.expand_path("lib", __dir__)
 # Maintain your gem's version:
 require "foreman_discovery/version"
 
+Dir['locale/**/*.po'].each do |po|
+  mo = po.sub(/foreman_discovery\.po$/, 'LC_MESSAGES/foreman_discovery.mo')
+  warn "WARNING: File #{mo} does not exist, generate with 'make all-mo'!" unless File.exist?(mo)
+  warn "WARNING: File #{mo} outdated, regenerate with 'make all-mo'" if File.mtime(po) > File.mtime(mo)
```

Comment:
This causes a build failure:
```
[!] There was an error while loading `foreman_discovery.gemspec`: No such file or directory @ rb_file_s_mtime - locale/cs_CZ/LC_MESSAGES/foreman_discovery.mo. Bundler cannot continue.
```

I think the line above it (`File.exist?`) should stop processing.